go.mod

```diff
@@ -6,20 +6,22 @@ retract [v1.0.0, v1.0.1]
 
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/ipfs/go-blockservice v0.1.2
+	github.com/gopherjs/gopherjs v0.0.0-20190430165422-3e4dfb77656c // indirect
+	github.com/ipfs/go-blockservice v0.1.4
 	github.com/ipfs/go-cid v0.0.7
 	github.com/ipfs/go-cidutil v0.0.2
 	github.com/ipfs/go-datastore v0.4.5
-	github.com/ipfs/go-ipfs-blockstore v0.1.0
+	github.com/ipfs/go-fetcher v1.3.0
+	github.com/ipfs/go-ipfs-blockstore v0.1.4
 	github.com/ipfs/go-ipfs-blocksutil v0.0.1
 	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
 	github.com/ipfs/go-ipfs-routing v0.1.0
 	github.com/ipfs/go-ipld-cbor v0.0.3
-	github.com/ipfs/go-ipld-format v0.0.2
-	github.com/ipfs/go-log v0.0.1
-	github.com/ipfs/go-merkledag v0.2.3
+	github.com/ipfs/go-log v1.0.4
 	github.com/ipfs/go-verifcid v0.0.1
-	github.com/libp2p/go-libp2p-core v0.2.2
-	github.com/libp2p/go-libp2p-testing v0.1.0
-	github.com/multiformats/go-multihash v0.0.13
+	github.com/ipld/go-ipld-prime v0.9.1-0.20210402181957-7406578571d1
```

Comment:
Why not a released version of go-ipld-prime?

Reply:
will update when we do the final merge push once the whole stack is approved